app.yaml

```diff
@@ -4,7 +4,7 @@ threadsafe: true
 
 handlers:
 - url: /ajax/.*
-  script: main.app
+  script: main.APP
```

Comment:
Why this change? I think lowercase is fine...

Reply:
I changed the main.py `app` class to `APP` to comply with Pylint constant variable rule (all uppercase) so it has to be updated here in the app.yaml.

Reply:
Aite, I put things back to where it was and added override Pylint rule.